packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -271,3 +281,76 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   componentId?: string;
 }
+
+export type ITextFieldStyleProps = Required<Pick<ITextFieldProps, 'theme'>> &
+  Pick<
```

Comment:
wonder how these props show up in documentation.

Reply:
My guess is that it's ignored.

Reply:
I fully expected it to not show up but I would rather optimize for doing the right thing here and improve the parser as a systemic fix. But at the same time, I don't want to unnecessarily invest in the parser atm given API extractor and TSDoc especially given all the other burning problems we have :)